assets/js/modules/analytics/datastore/accounts.js

```diff
@@ -161,31 +161,33 @@ export const actions = {
 		invariant( profileName, 'profileName is required.' );
 		invariant( timezone, 'timezone is required.' );
 
-		let accountTicket;
+		let response,
+			error;
 		yield {
 			payload: {},
 			type: START_FETCH_CREATE_ACCOUNT,
 		};
 
 		try {
-			accountTicket = yield {
+			response = yield {
 				payload: { accountName, propertyName, profileName, timezone },
 				type: FETCH_CREATE_ACCOUNT,
 			};
-
+			const { accountTicket } = response;
```

Comment:
This doesn't work I think? The `response` _is_ the `accountTicket` I thought, but here you're destructuring it. I think the `response` can simply be passed to `receiveCreateAccount` below.

Reply:
Oh, right - fixing.

Reply:
Corrected in https://github.com/google/site-kit-wp/pull/1390/commits/8d2616d5a8ee0eb54c9bb104f154314dabc850bc.